src/components/test/NoticeBox.test.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * @jest-environment jsdom
+ */
+/* global expect */
+
+/**
+ * External dependencies
+ */
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+
+/**
+ * Internal dependencies
+ */
+import {NoticeBox} from "../NoticeBox";
+
+test("notice box outputs the correct messages", () => {
+  // test without being indexed
```

Comment:
would it be better if we use describe+test, where when the tests fail, we'll be able to know directly what has gone wrong. https://jestjs.io/docs/api#describename-fn

Reply:
And the same applies to the following tests.

Reply:
Ah, that looks so much better. I didn't know about using describe+test like this, that's really helpful and looking much better already 😄